src/main/java/org/openelisglobal/storage/controller/StorageLocationRestController.java

```diff
@@ -1156,6 +1184,9 @@ private Map<String, Object> entityToMap(Object entity) {
             map.put("capacityLimit", device.getCapacityLimit());
             map.put("active", device.getActive());
             map.put("fhirUuid", device.getFhirUuidAsString());
+            map.put("ipAddress", device.getIpAddress());
```

Comment:
This is fine but in general its better to use strongly typed objects rather than loosely typed maps for response objects